client/layout/masterbar/style.scss

```diff
@@ -38,7 +43,7 @@ $autobar-height: 20px;
 	display: flex;
 		align-items: center;
 	position: relative;
-	color: $white;
+	@include theme( color, --masterbar-color );
```

Comment:
💯 Love it! Abstracting the additional complexity away and doing the magic behind the scenes would not only increase the developer experience but more importantly enable us to simply pull the plug on the post-processing step once the fallback isn't necessary anymore. Great input, I'll look into it. 👍